concepts/observability-configurable-logging/07-multiple-logpipeline-investigation.md

```diff
@@ -0,0 +1,136 @@
+# Objective
+
+The telemetry operator comes with a pipeline configured to push logs to Loki. Additionally, Kyma users can define their own pipeline and push the logs to their own logging backend.
+
+This document investigates the side effects that may come up when, for example, the logging backend defined by  user stops working.
+
+## Setup
+
+The investigated configuration uses the `filesystem buffering`. When one of the log pipeline outputs fails, the logs are buffered on the file system. In contrast, the `in-memory` buffer has the disadvantage that if the Fluent Bit Pod is restarted, logs can be lost. During the test, we referred to the Fluent Bit knowledge article about [Buffering & Storage](https://docs.fluentbit.io/manual/administration/buffering-and-storage).
```

Comment:
Let's use "file system" or "filesystem" coherently

Reply:
We should also mention that Fluent Bit behaves differently when the filesystem storage is active:
- If the non-persistent memory storage is used, the input is paused if an output stops processing.
- If the filesystem storage is used, the input is never stopped and the filesystem utilized if logs cannot be send to the output. Logs are dropped if the configured limit of the filesystem storage is reached.

Reply:
fixed